pkg/types/types.go

```diff
@@ -89,10 +89,11 @@ type DeviceSelectors struct {
 // NetDeviceSelectors contains network device related selectors fields
 type NetDeviceSelectors struct {
 	DeviceSelectors
-	PfNames     []string `json:"pfNames,omitempty"`
-	LinkTypes   []string `json:"linkTypes,omitempty"`
-	DDPProfiles []string `json:"ddpProfiles,omitempty"`
-	IsRdma      bool     // the resource support rdma
+	PfNames      []string `json:"pfNames,omitempty"`
+	LinkTypes    []string `json:"linkTypes,omitempty"`
+	DDPProfiles  []string `json:"ddpProfiles,omitempty"`
+	IsRdma       bool     // the resource support rdma
+	NeedVhostNet bool     // share vhost-net along the selected pf
```

Comment:
I'm little confused by this comment here. Does vhost-net depend on a PF?

Reply:
It's a typo, it should be 'vf'.

Reply:
It's a typo, it should be 'ressource' : // share vhost-net along the selected ressource